tests/assets/minimal-cluster.conf

```diff
@@ -1,4 +1,5 @@
 # Minimal configuration for testing.
+databases 1
```

Comment:
This can be removed, in the future we will support multi dbs.
```suggestion
```

I added this because it will print the warning in local.
```
    /* in case cluster mode is enabled dbnum must be 1 */
    if (server.cluster_enabled && server.dbnum > 1) {
        serverLog(LL_WARNING, "WARNING: Changing databases number from %d to 1 since we are in cluster mode",
                  server.dbnum);
        server.dbnum = 1;
    }
```